internal/engine/upper_test.go

```diff
@@ -2819,7 +2820,7 @@ func newTestFixture(t *testing.T) *testFixture {
 	tvc := NewTiltVersionChecker(func() github.Client { return ghc }, tiltVersionCheckTimerMaker)
 
 	subs := ProvideSubscribers(fakeHud, pw, sw, plm, pfc, fwm, bc, ic, cc, dcw, dclm, pm, sm, ar, hudsc, sc, tvc, tas, ewm)
-	ret.upper = NewUpper(ctx, st, subs)
+	ret.upper = NewUpper(ctx, st, subs, token.Token(""))
```

Comment:
I might make this `"unit test token"` or something, just in case at some point someone accidentally changes code leading to a test making a real request and we end up with a bunch of unit test requests in our logs / analytics